astroquery/esa/hubble/core.py

```diff
@@ -415,15 +504,16 @@ def _query_tap_target(self, target):
             target_result = target_response.json()['data'][0]
             ra = target_result['RA_DEGREES']
             dec = target_result['DEC_DEGREES']
-            return ra, dec
+            return SkyCoord(ra=ra, dec=dec, unit="deg")
+            # return ra, dec
```

Comment:
don't leave commented out code lines, remove them. If any of these are needed for bug hunting purposes, they are in the git history.